region/maridia/inner-green.json

```diff
@@ -1420,8 +1794,23 @@
                   "notable": false,
                   "requires": [
                     "Gravity",
-                    "canConsecutiveWalljump"
-                  ]
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      "SpaceJump"
+                    ]},
+                    {"or": [
+                      {"enemyDamage": {
+                        "enemy": "Menu",
+                        "type": "contact",
+                        "hits": 1
+                      }},
+                      "ScrewAttack",
+                      "canStaggeredWalljump",
+                      "Charge"
+                    ]}
+                  ],
+                  "note": "The swarm of Menus will attack Samus at the top of the room.",
+                  "devNote": "The Menus prevent a reliable IBJ."
```

Comment:
ibj with a PB?

Reply:
I tried it with a PB and couldn't clear them.  You have to lay it pretty low and that means you can't kill the Menus up high who will swoop down right after.